README.md

```diff
@@ -1543,6 +1545,47 @@ FLAG DESCRIPTIONS
     scripts), you can use this flag to easily get the IDs of created resources for further processing.
 ```
 
+## `mw app upload [INSTALLATION-ID]`
```

Comment:
If I can _upload_ an app, it would be great to _download_ it as well 

This is only a feature idea and should be implemented in a follow up pull request.

Reply:
That command already exists,[^1] added in #173 😉 

[^1]: https://github.com/mittwald/cli?tab=readme-ov-file#mw-app-download-installation-id